docs/Formula-Cookbook.md

```diff
@@ -661,25 +661,25 @@ Generally we'd rather you were specific about what files or directories need to
 
 #### Variables for directory locations
 
-| Name                  | Default                                        | Example                                           |
-|-----------------------|------------------------------------------------|---------------------------------------------------|
-| **`HOMEBREW_PREFIX`** | `/usr/local`                                   |                                                   |
-| **`prefix`**          | `#{HOMEBREW_PREFIX}/Cellar/#{name}/#{version}` | `/usr/local/Cellar/foo/0.1`                       |
-| **`opt_prefix`**      | `#{HOMEBREW_PREFIX}/opt/#{name}`               | `/usr/local/opt/foo`                              |
-| **`bin`**             | `#{prefix}/bin`                                | `/usr/local/Cellar/foo/0.1/bin`                   |
-| **`doc`**             | `#{prefix}/share/doc/foo`                      | `/usr/local/Cellar/foo/0.1/share/doc/foo`         |
-| **`include`**         | `#{prefix}/include`                            | `/usr/local/Cellar/foo/0.1/include`               |
-| **`info`**            | `#{prefix}/share/info`                         | `/usr/local/Cellar/foo/0.1/share/info`            |
-| **`lib`**             | `#{prefix}/lib`                                | `/usr/local/Cellar/foo/0.1/lib`                   |
-| **`libexec`**         | `#{prefix}/libexec`                            | `/usr/local/Cellar/foo/0.1/libexec`               |
-| **`man`**             | `#{prefix}/share/man`                          | `/usr/local/Cellar/foo/0.1/share/man`             |
-| **`man[1-8]`**        | `#{prefix}/share/man/man[1-8]`                 | `/usr/local/Cellar/foo/0.1/share/man/man[1-8]`    |
-| **`sbin`**            | `#{prefix}/sbin`                               | `/usr/local/Cellar/foo/0.1/sbin`                  |
-| **`share`**           | `#{prefix}/share`                              | `/usr/local/Cellar/foo/0.1/share`                 |
-| **`pkgshare`**        | `#{prefix}/share/foo`                          | `/usr/local/Cellar/foo/0.1/share/foo`             |
```

Comment:
Did you intend to remove `pkgshare`? Can you re-add it? Also, as @Bo98 mentioned ideally could you do all of these: https://github.com/Homebrew/brew/blob/9a48d24545eac467f641be03a7c3762c241fa243/Library/Homebrew/formula.rb#L877-L915

Thanks ❤️

Reply:
> Did you intend to remove `pkgshare`?

No, not sure what happened there… had to disable my markdown table editor package which kept interfering.

> Can you re-add it? Also, as @Bo98 mentioned ideally could you do all of these:

Done.